### PR TITLE
feat: add full privacy signature

### PR DESCRIPTION
**Pull request by gnarco**

### CODE REVIEW

**Review by alexishecf — APPROVED** [review]

File: public/version7/ArianeeProductCertificate-i18n.json [hunk 1]
```diff
@@ -0,0 +1,2235 @@
+{
+  "$id": "https://cert.arianee.org/version6/ArianeeProductCertificate-i18n.json",
```
Comment (stefdelec) [inline public/version7/ArianeeProductCertificate-i18n.json:2]: version7

**Review by stefdelec — CHANGES_REQUESTED** [review]

File: public/version7/ArianeeProductCertificate-i18n.json (hunk 1 above)
Comment (gnarco) [reply public/version7/ArianeeProductCertificate-i18n.json]: fixed

File: public/version2/ArianeeMessage-i18n.json [hunk 2]
```diff
@@ -0,0 +1,296 @@
+{
+    "$id": "https://cert.arianee.org/version1/ArianeeMessage-i18n.json",
```
Comment (stefdelec) [inline public/version2/ArianeeMessage-i18n.json:2]: version2

File: public/version2/ArianeeEvent-i18n.json [hunk 3]
```diff
@@ -0,0 +1,400 @@
+{
+  "$id": "https://cert.arianee.org/version1/ArianeeEvent-i18n.json",
```
Comment (stefdelec) [inline public/version2/ArianeeEvent-i18n.json:2]: version2

**Review by stefdelec — CHANGES_REQUESTED** [review]

**Review by stefdelec — APPROVED** [review]

**Review by stefdelec — APPROVED** [review]

**Pull request merged**